### PR TITLE
feat(codegen): more visible 'unmanaged' comment

### PR DESCRIPTION
**Pull request by raboof**
Refs #549

I'd like a review on this before doing all the other
classes as well ;)

### CODE REVIEW

File: codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/ActionServiceSourceGenerator.scala [hunk 1]
```diff
@@ -137,13 +137,13 @@ object ActionServiceSourceGenerator {
...
-    s"""|$unmanagedComment
-        |
-        |package $packageName;
+    s"""|package $packageName;
         |
         |$imports
         |
-        |/** An action. */
+        |/**
+        | * An action.$unmanagedComment
```
Comment (patriknw) [inline codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/ActionServiceSourceGenerator.scala:145]: I think that comment is more a developer description than something that belongs in api doc section. Could we have it as a separate comment (as it was before, or with `//` style) right after the imports but not inside the api doc?

Comment (raboof) [reply codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/ActionServiceSourceGenerator.scala]: works for me, updated

**Review by johanandren — APPROVED** [review]

**Review by octonato — APPROVED** [review]

**Review by patriknw — APPROVED** [review]
LGTM when all of them have been updated

**Comment by pvlugter** [comment]
👍🏼

**Pull request merged**